src/redux/actions/nearbyActions.js

```diff
@@ -17,18 +18,18 @@ export const ControllerValues = {
 
 
 export function Update_Store(slider, value) {
-	console.log("test")
-	console.log(slider)
+  console.log("test")
```

Comment:
We probably shouldn't have debug console.logs in master

Reply:
They've been there a while honestly, we can drop them easily though,

Reply:
please do that mr zbuttons. thanks

Reply:
I have a big commit coming to reducers/nearby and will remove these in that pull request.

Reply:
lol, sounds good

Reply:
:point_up: